src/MongoDB.AspNetCore.OData/MongoEnableQueryAttribute.cs

```diff
@@ -73,6 +81,12 @@ public override IQueryable ApplyQuery(IQueryable queryable, ODataQueryOptions qu
         return queryable;
     }
 
+    private Expression GetRewrittenExpression(IQueryable queryable)
```

Comment:
Rename the method to `RewriteExpression` and change return type to `IQueryable`.

Reply:
Done.